internal/controller/accountiam_controller.go

```diff
@@ -897,5 +899,11 @@ func (r *AccountIAMReconciler) createOrUpdate(ctx context.Context, obj *unstruct
 func (r *AccountIAMReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&operatorv1alpha1.AccountIAM{}).
+		Owns(&appsv1.Deployment{}).
+		Owns(&corev1.Secret{}).
+		Owns(&corev1.ConfigMap{}).
+		Owns(&corev1.Service{}).
+		Owns(&corev1.ServiceAccount{}).
+		Owns(&routev1.Route{}).
```

Comment:
the rbac resources are missing, right? Role and Rolebinding?

Reply:
Yes, I have added RoleBinding to be watched by the controller, but the Role keeps changing, which triggers reconciliation continuously. So, I temporarily commented out the resources that change during each reconciliation and need to figure out the cause